tests/test-ec2dhcp-down-ipv6.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the MIT License. See the LICENSE accompanying this file
+# for the specific language governing permissions and limitations under
+# the License.
+
+. ./common.sh
+# test params files should define a dhclient environment
+
+if command -v dhclient6_env_down > /dev/null 2>&1; then
+    dhclient6_env_down
+    . ../ec2dhcp.sh
```

Comment:
Based on an earlier comment, shouldn't `ec2dhcp.sh` be deleted and thus this is an error?

Reply:
the deleted file is the `tests/test-ec2dhcp.sh` script. The top level `ec2dhcp.sh` script, which is installed as part of the RPM, most definitely still exists.